assignments/closure.js

```diff
@@ -1,14 +1,27 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+function sayIt(){
+  let name = "Csilla";
+  function magyar(){
+    console.log("Szep jo napot"+ name);
+  }
+  magyar();
+}
+console.log(sayIt());
 
 // ==== Challenge 2: Create a counter function ====
 const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
-};
+
 // Example usage: const newCounter = counter();
 // newCounter(); // 1
 // newCounter(); // 2
+let count=0;
```

Comment:
I know you are still reading and researching before finishing this one. If you need more resources or anything, let me know.